generate.sh

```diff
@@ -1,5 +1,8 @@
 ROOTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" 
 
+# Flags scripts to use experiment branch for the samples
+export SAMPLE_BRANCH=experiment
```

Comment:
Should we set the env in all 3 scripts being called in `generate.sh` since they all look for that env as well? If I set it in just the ai lab samples I believe it will try and pull in all the gitops and tekton items